tests/legacy/test_data/monsieurcuisine.testjson

```diff
@@ -0,0 +1,891 @@
+{
+    "code": 0,
+    "message": "A recipe is returned",
+    "data": {
+        "recipe": {
+            "id": 1741327,
+            "translationId": 417065,
+            "language": "de-DE",
+            "author": {
+                "id": 215,
+                "name": "Monsieur Cuisine Official",
+                "avatar": "https://mc-recipe-web-prod.storage.googleapis.com/wp-content/uploads/2021/11/15171610/avatar1636967684.png"
+            },
+            "name": "Paprika-Nudel-Auflauf mit Hackfleisch",
+            "status": "publish",
+            "deviceType": "MC2.0",
+            "deviceTypes": [
+                "MC2.0",
+                "MC3.0"
+            ],
+            "publishedDate": "2022-08-29 13:47:53",
+            "createdDate": "2022-08-29 13:47:53",
+            "lastUpdated": "2024-03-06 12:52:22",
+            "thumbnail": {
+                "landscape": "https://mc-web-content-cdn.tecpal.com/wp-content/uploads/2022/08/29214717/paprika-nudelauflauf-mit-hackfleisch-1.jpg",
+                "portrait": null
+            },
+            "complexity": "Einfach",
+            "complexityLevel": 1,
+            "preparationDuration": 70,
+            "duration": 70,
+            "rating": 4.5,
+            "totalRating": 1013,
+            "popularity": 2.37365,
+            "source": "official",
+            "media": null,
+            "url": "https://www.monsieur-cuisine.com/de/recipe/paprika-nudel-auflauf-mit-hackfleisch",
+            "categories": [
```

Comment:
It would be nice to [add the `category` information](https://github.com/hhursev/recipe-scrapers/blob/4da3a6493cebe91c85378a2307645006b4fb17a9/docs/in-depth-guide-scraper-functions.md#category---str) since we receive data about it in the JSON; one note about this is that we usually translate list-of-categories into comma-separated values ("Snacks, Backen, Abendessen, ZauberTopf" in this case probably).

Reply:
Okay.

Reply:
Thank you!